src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/EnumWithFlagsAttribute.Fixer.cs

```diff
@@ -75,9 +75,11 @@ private static SyntaxNode RemoveFlagsAttribute(SyntaxGenerator generator, Semant
             }
 
             AttributeData flagsAttribute = enumType.GetAttributes().First(a => Equals(a.AttributeClass, flagsAttributeType));
-            SyntaxNode attributeNode = flagsAttribute.ApplicationSyntaxReference.GetSyntax(cancellationToken);
+            SyntaxNode? attributeNode = flagsAttribute.ApplicationSyntaxReference.GetSyntax(cancellationToken);
```

Comment:
Do we have a repro case where this is null? I am not sure if this can ever be null.

Reply:
If I have correctly understood your comment about base type, then for an enum it seems impossible so we don't need to check this. Is it worth adding a `RoslynDebug.Assert`?